### DISCUSSION TITLE
Segmentation fault on mac os

### DISCUSSION THREAD

**Discussion by toludoyin** [psycopg/psycopg2#1565] [Q&A]
I am writing a python script to query data from postgreSQL on vscode

After installing psycopg2-binary, I got a segmentation fault error. Is there something I'm not doing right?

I tried to reinstall python from brew and reinstalled venv as mentioned [here](https://github.com/psycopg/psycopg2/discussions/1538) by
[arogozhnikov](https://github.com/arogozhnikov) but the error still persists.

**Comment by dvarrazzo** [comment]
Try installing the non -binary version.

**Reply by toludoyin** [reply]
I uninstalled the binary version and installed the non-binary version, tried to run my code, and got "ModuleNotFoundError: No module named 'psycopg2' "
